src/twisted/web/http.py

```diff
@@ -319,11 +323,26 @@ class _MultiPartParseException(Exception):
     """
 
 
-def _getMultiPartArgs(content: bytes, ctype: bytes) -> dict[bytes, list[bytes]]:
+@dataclass
+class FileUpload:
+    """
+    A file that was uploaded to the web server as part of multipart request.
+    """
+
+    data: BinaryIO
+    name: str
+    filename: str
+    contentType: str
```

Comment:
It's a bit sad that it is 2024 and we do not have a structured object representing a MIME type anywhere in either the stdlib or Twisted, but it would be a bridge too far to make that your problem right now :)